examples/chip-tool/commands/common/ModelCommand.cpp

```diff
@@ -28,8 +28,8 @@
 using namespace ::chip;
 using namespace ::chip::DeviceController;
 
-#define CHECK_MESSAGE_LENGTH(rv)                                                                                                   \
-    VerifyOrExit(rv, ChipLogError(chipTool, "%s: Unexpected response length: %d", __FUNCTION__, messageLen));
+#define ZCL_DEFAULT_RESPONSE_COMMAND_ID 0x0b
```

Comment:
nit: consider C++ 

```
namespace {
   constexpr uint8_t kDefaultResponseCommandId = 0x0b;
   constexpr uint8_t kReadAttributesResponseCommandId = 0x01;
}
```

or if these are enums we could have a CommandId enum.

#define does not feel as C++